pkg/auth/providers/common/ldap/ldap_util.go

```diff
@@ -57,6 +57,14 @@ func NewLDAPConn(servers []string, TLS bool, port int64, connectionTimeout int64
 	return lConn, nil
 }
 
+func Connect(config *v3.LdapConfig, caPool *x509.CertPool) (*ldapv2.Conn, error) {
+	servers := config.Servers
+	TLS := config.TLS
+	port := config.Port
+	connectionTimeout := config.ConnectionTimeout
+	return NewLDAPConn(servers, TLS, port, connectionTimeout, caPool)
```

Comment:
nit: we can call NewLDAPConn(config.Servers, config.TLS, config.port, config.connectionTimeout, caPool) directly instead of assigning to new variables?

Reply:
yup, totally can good idea. wasn't my func initially but will do that.